src/main/resources/db/changelog/v.0.1/db.changelog-internship.yaml

```diff
@@ -18,13 +18,13 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: inship_start_date
-                  type: TIMESTAMP
+                  type: DATE
```

Comment:
we must not change existing changesets

Reply:
Ok, in the future I won't change it. Can I leave it now?

Reply:
No. 
Merge the latest develop branch. It already contains this changes.

Reply:
Done